fleet_adapter_mir/fleet_adapter_mir/fleet_adapter_mir.py

```diff
@@ -286,18 +173,20 @@ def main(argv=sys.argv):
 
     # INIT RCL ================================================================
     rclpy.init(args=argv)
-    adpt.init_rclcpp()
+    rmf_adapter.init_rclcpp()
     args_without_ros = rclpy.utilities.remove_ros_args(argv)
 
     parser = argparse.ArgumentParser(
-        prog="fleet_adapter_mir",
+        prog="cgh_fleet_adapter_mir",
         description="Configure and spin up fleet adapters for MiR 100 robots "
                     "that interface between the "
                     "MiR REST API, ROS2, and rmf_core!")
     parser.add_argument("-c", "--config_file", type=str, required=True,
                         help="Input config.yaml file to process")
     parser.add_argument("-n", "--nav_graph", type=str, required=True,
                     help="Path to the nav_graph for this fleet adapter")
+    parser.add_argument("-a", "--actions", type=str, required=False, default='',
```

Comment:
As we move towards a modular plugin design, I think it would make sense to let this be a list of action files instead of only one action file. Then users can add their own custom actions on top of the ones we provide by writing their own additional .json files.

Reply:
This `actions` argument takes in the base rmf missions (path to `rmf_missions.json` [here](https://github.com/osrf/fleet_adapter_mir/tree/xiyu/efc_fleet_adapter_mir/missions)) required for default tasks such as patrol, localize and charging. Any perform action-related missions will be parsed in from the config as a [field under each plugin](https://github.com/osrf/fleet_adapter_mir/blob/xiyu/efc_fleet_adapter_mir/configs/mir_config.yaml#L111).

I realized naming it `actions` is extremely misleading. Actions have a different meaning associated with MiR missions (they are the building blocks of the missions). I've changed it to `rmf_missions`.